src/node_crypto.cc

```diff
@@ -641,6 +641,17 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
       min_version = TLS1_2_VERSION;
       max_version = TLS1_2_VERSION;
       method = TLS_client_method();
+    } else if (strcmp(*sslmethod, "TLSv1_3_method") == 0) {
```

Comment:
if we want as a feature, should be standalone, they are deliberately not implemented ATM

Reply:
Actually, I think this was only needed in the original QUIC modifications to OpenSSL. I don't think we need these with the Boring APIs.